### PR TITLE
Issue 2730 - Fix value of notification key "inactive" was ignored

### PR DESCRIPTION
**Pull request by andreynovikov**

### CODE REVIEW

File: sitescripts/notifications/parser.py [hunk 1]
```diff
@@ -67,7 +70,9 @@ def _parse_notification(data, name):
     is_variant = current != notification
 
     if key == "inactive" and not is_variant:
-      current["inactive"] = True
+      if value.lower() not in BOOLEAN_STATES:
```
Comment (snoack) [inline sitescripts/notifications/parser.py]: I think it's safe to simply consider any value that isn't in `('', '0', 'false', 'no', 'off'')` truthful. That would simplify the logic a bit.

**Comment by andreynovikov** [comment]
I do not have knowledge how to modify pull request. :)

**Comment by snoack** [comment]
Just push to the branch you created the pull request from.

**Comment by andreynovikov** [comment]
Cool, have done it.

**Comment by snoack** [comment]
Thanks! LGTM and merged.

**Pull request closed without merging**